thrimshim/thrimshim/main.py

```diff
@@ -656,15 +659,16 @@ def validate_template(new_template):
 	for extra in extras:
 		del new_template[extra]
```

Comment:
Wouldn't this mean we can *never* update the image due to it being removed as a "non-required" field?

Reply:
If `image` is a field in the `new_template` it gets added to the list of columns, even if `require_image` is `False`.